addons/point_of_sale/point_of_sale.py

```diff
@@ -1140,11 +1140,14 @@ def insert_data(data_type, values):
             cur = order.pricelist_id.currency_id
             for line in order.lines:
                 tax_amount = 0
-                taxes = []
-                for t in line.product_id.taxes_id:
-                    if t.company_id.id == current_company.id:
-                        taxes.append(t)
-                computed_taxes = account_tax_obj.compute_all(cr, uid, taxes, line.price_unit * (100.0-line.discount) / 100.0, line.qty)['taxes']
+                if line.tax_ids:
+                    tax_ids = line.tax_ids
+                else:
+                    tax_ids = []
+                    for t in line.product_id.taxes_id:
+                        if t.company_id.id == current_company.id:
```

Comment:
Not your code, but do we have any idea why we check the taxes' company in this case but not for the same case in `action_invoice`?